omnigibson/object_states/link_based_state_mixin.py

```diff
@@ -43,12 +52,13 @@ def _default_link(self):
     def initialize_link_mixin(self):
         assert not self._initialized
 
-        self._link = None
         # TODO: Extend logic to account for multiple instances of the same metalink? e.g: _0, _1, ... suffixes
         for name, link in self.obj.links.items():
             if self.metalink_prefix in name:
-                self._link = link
-                break
+                self._links[name] = link
+
+        if len(self._links) > 0:
+            self._link = list(self._links.values())[0]
```

Comment:
hmmm...I feel like the current paradigm of having `link` and `links` is a bit confusing. I wonder if we can unify? maybe we make `link` --> `default_link` or something (or just deprecate `link` completely, which is probably the proper solution long-term, but requires another refactoring of all other link-based states). happy to discuss.

Reply:
Discussed in person. We will address this in the next PR that properly allows object states to use more than one meta link per type.